domain/auth/auth.go

```diff
@@ -96,7 +97,9 @@ func CtxWithAccessToken(ctx context.Context, at AccessToken) context.Context {
 // read (GET) the resource at the /ping path. This is obviously
 // completely bogus right now, eventually need to look into something
 // for ACL/RBAC
-type Authorizer struct{}
+type Authorizer struct {
+	Enforcer *casbin.Enforcer
+}
 
 // Authorize authorizes a subject (user) can perform a particular
 // action on an object. e.g. gilcrest can read (GET) the resource
```

Comment:
I think we should update the Authorize method comment to:
and the Authorize method comment to:

// Authorize authorizes that a subject (user.User) can perform a
// particular action on an object. e.g. subject otto.maddox711@gmail.com
// can read (GET) the object (resource) at the /api/v1/movies path.
// Casbin is setup to use an RBAC (Role-Based Access Control) model
// Users with the `admin` role can **write** (GET, PUT, POST, DELETE).
// Users with the `user` role can only **read** (GET)